### ISSUE TITLE
a11y: organize testing by a blind tester

### ISSUE THREAD

**Issue by aristov** [bem/bem-components#1684]
We need this to receive a more natural feedback and to detect unnoticed problems on the [a11y improvement work](https://github.com/bem/bem-components/issues/1206) was done recently.

The [described known issues](https://github.com/bem/bem-components/pull/1665#issuecomment-142277774) should be taken into account.

cc @tadatuta

**Comment by aristov** [comment]
cc @deeonis
